library/prolog_breakpoints.pl

```diff
@@ -60,19 +61,28 @@
 other hooks the opportunity to react.
 */
 
+:- meta_predicate set_breakpoint(+, +, +, +, 0, -).
+
 %!  set_breakpoint(+File, +Line, +Char, -Id) is det.
 %!  set_breakpoint(+Owner, +File, +Line, +Char, -Id) is det.
+%!  set_breakpoint(+Owner, +File, +Line, +Char, :Cond, -Id) is det.
```

Comment:
I had expected an interface set_breakpoint_condition(+Id, :Cond).   That would make the human interface easier: set a breakpoint by in the gui, then select the breakpoint and add the condition.   It would also facilitate changing the condition if you discover that the breakpoint is still called too often.

Reply:
I see, that makes sense, I've added `set_breakpoint_condition/2` in https://github.com/SWI-Prolog/swipl-devel/pull/991/commits/8946aa5de2417e20446006f2fd1d3d209d61e17e
(Also rebased to track the latest commits in the master branch, hence the force-push)